nixos/default.nix

```
@@ -98,6 +98,8 @@ in {
 
         environment = serviceEnvironment;
 
+        unitConfig = { RequiresMountsFor = "/home/${username}"; };
```

This is a good idea, thanks for fixing!

Unfortunately you cannot assume that the home directory is under `/home`, however. Better use something like
```suggestion
        unitConfig = { RequiresMountsFor = usercfg.home.homeDirectory; };
```

---

Changed it, Thanks